.github/workflows/linters.yml

```diff
@@ -25,5 +25,5 @@ jobs:
     steps:
       - uses: ansible-network/github_actions/.github/actions/checkout_dependency@main
 
-      - name: Run ansible-lint
-        uses: ansible/ansible-lint@v6.21.0
+      - name: run-ansible-lint
```

Comment:
Why the change of step name?

Reply:
Replaced it with new task from https://github.com/marketplace/actions/run-ansible-lint

Reply:
<img width="768" alt="image" src="https://github.com/user-attachments/assets/39ec8f64-a464-4eab-9898-bd3d64069a14">